foundry.toml

```diff
@@ -9,15 +9,16 @@ multiline_func_header = 'params_first'
 sort_imports = true
 
 [profile.default]
-solc_version = '0.8.25'
```

Comment:
bruh!

Reply:
does it work with `0.8.23` btw?

Reply:
0.8.23 works, when not specifying in the toml, that's what forge uses here -> I'll try to get back to 0.8.25 but shangai (as it can be done as an arg for crytic-compile, this would then only leave the mutex as difference with dev)